README.md

```diff
@@ -39,7 +39,7 @@
 </p>
 
 <p align="center">
-  <a href=https://github.com/danny-avila/LibreChat/blob/main/.github/TRANSLATION.md">
+  <a href="https://github.com/danny-avila/LibreChat/blob/main/.github/TRANSLATION.md">
```

Comment:
change to: https://www.librechat.ai/docs/translation

Reply:
and this file can be removed 😉
https://github.com/danny-avila/LibreChat/blob/main/.github/TRANSLATION.md